sql/src/main/java/io/crate/execution/engine/collect/collectors/LuceneOrderedDocCollector.java

```diff
@@ -132,10 +132,7 @@ public void close() {
         TopFieldCollector topFieldCollector = TopFieldCollector.create(
             sort,
             batchSize,
-            true,
-            doDocsScores,
-            doDocsScores,
-            false           // trackTotalHits - we don't use the number of total hits
+            batchSize
```

Comment:
as we're not interested in the total hits, we should pass `1` here to not slow down the query. also a comment similar to the removed one would be great.